Common/src/main/resources/META-INF/mods.toml

```diff
@@ -16,21 +16,21 @@ Shows a simple representation of the screen players are currently looking at
 
 '''
 # A dependency - use the . to indicate dependency for a specific modid. Dependencies are optional.
-[[dependencies.forge]] #optional
+[[dependencies.publicguiannouncement]] #optional
     # the modid of the dependency
     modId="forge" #mandatory
     # Does this dependency have to exist - if not, ordering below must be specified
     mandatory=true #mandatory
     # The version range of the dependency
-    versionRange="[41,)" #mandatory
+    versionRange="[41.0.94,)" #mandatory
     # An ordering relationship for the dependency - BEFORE or AFTER required if the relationship is not mandatory
     ordering="NONE"
     # Side this dependency is applied on - BOTH, CLIENT or SERVER
     side="BOTH"
 # Here's another dependency
-[[dependencies.minecraft]]
+[[dependencies.publicguiannouncement]]
     modId="minecraft"
     mandatory=true
-    versionRange="[1.19)"
+    versionRange="[1.19,1.20)"
```

Comment:
please only leave one version in, so any future versions may be accepted equally. 
1.19 is more then enough. 1.20 isn't out yet and a superfluous addition.
1.19) will equally allow for 1.20 to be accepted, if ever they should be compatible

Reply:
`[1.19)` doesn't work[^1], it just didn't cause an error before because the dependencies list had the wrong mod id, which meant forge wasn't checking it. looking at [the spec for the version ranges](https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html), it looks like it needs have a comma added `[1.19,)` to be valid, but that also fails[^2] (though this time i'm not really sure why). `[1.19,1.20)` was what the forge mdk's sample mods.toml had, and it worked, so I went with that.

[^1]: crashes with `FATAL` ... `Failed to parse version spec [1.19)`
[^2]: fails with `ModLoadingException: The Mod File `...`Public GUI Announcement-common-1.19-4.0.0.0.jar has mods that were not found`